package.json

```diff
@@ -1,19 +1,20 @@
 {
   "name": "@schibstedspain/react-components",
-  "version": "1.2.0",
+  "version": "1.0.0",
```

Comment:
Do you decrease the version?

Reply:
The version of the root package is used for nothing. :D